prisma/schema.prisma

```diff
@@ -321,15 +321,15 @@ enum ApplicantStatus {
   ACTIVE
   INACTIVE
 
-  @@map("applicantstatus")
+   @@map("applicantstatus")
```

Comment:
Not sure why these spaces are being added. Could you please run `npx prisma format`?